Firestore/Source/Core/FSTEventManager.mm

```diff
@@ -223,13 +84,17 @@ @interface FSTEventManager () <FSTSyncEngineDelegate>
 - (instancetype)initWithSyncEngine:(FSTSyncEngine *)syncEngine NS_DESIGNATED_INITIALIZER;
 
 @property(nonatomic, strong, readonly) FSTSyncEngine *syncEngine;
-@property(nonatomic, strong, readonly)
-    NSMutableDictionary<FSTQuery *, FSTQueryListenersInfo *> *queries;
 @property(nonatomic, assign) OnlineState onlineState;
 
 @end
 
-@implementation FSTEventManager
+@implementation FSTEventManager {
+  std::unordered_map<FSTQuery *,
```

Comment:
See my comment in the other PR re. potentially making this `objc::unordered_map` to reduce the number of template arguments.